src/lambdapi.ml

```diff
@@ -175,14 +175,14 @@ let keep_order : bool Term.t =
 
 (** Debugging and output options. *)
 
-let verbose : int Term.t =
+let verbose : int option Term.t =
   let doc =
     "Set the verbosity level to $(docv). A value smaller or equal to 0 will \
      disable all printing (on standard output). Greater numbers lead to more \
      and more informations being written to standard output. There is no \
      difference between the values of 3 and more."
   in
-  Arg.(value & opt int 1 & info ["verbose"; "v"] ~docv:"NUM" ~doc)
+  Arg.(value & opt (some int) None & info ["verbose"; "v"] ~docv:"NUM" ~doc)
```

Comment:
I think that just `opt int` would suffice here. But I am not exactly sure.

Reply:
Is the important thing to perform an `Option.iter` in `init`? If so, then the `some` is required to have on `option` type (I discovered cmdliner this morning, so I may be wrong).

Reply:
Oh yeah, you are right I think. I am still not completely used to it myself. I only had a serious look at it for my PR.

Reply:
Actually, I just remembered: if you don't use the `some` thingy` then you need to give the default verbosity value in the command line argument, which is not what we want since it is what was done before...

I checked, the default value is indeed set to `1` in `src/core/console.ml` so what you have here is perfect.